### PR TITLE
Create path-migrating-data-from-fast-to-cloud.md

### PR DESCRIPTION
**Pull request by jnyhuis**
Adding draft Path Document for data migration to S3

### CODE REVIEW

**Review by vortexing — COMMENTED** [review]
I have a few things here.
- Prereq's - how will they know what "understand" means?  Perhaps re-word to contextualize those links more. (also objectives is not the right word for object store)

- Notes comment - that will get old and stale and we'll never find it again.  It's best to not add any new information in a Pathway, but link to things that are more actively updated.  Consider moving these notes to a SciComp announce entry if you'd like to announce a policy change and then linking to that.

**Comment by jnyhuis** [comment]
OK, I have made the requested adjustments.
Thx -John N.

**Comment by chrisequalsdev** [comment]
- Instead of suggesting the Motuz API, it would be better to link to existing documentation on using the AWS cli 

- For the problems section, e-mailing scicomp is good but remove filing an issue in the Motuz repo.

File: _pathways/path-migrating-data-from-fast-to-cloud.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ - [Copying Files To Cloud Locations](https://sciwiki.fredhutch.org/compdemos/motuz/#copying-files-to-cloud-locations)
+
+Motuz can copy files into S3 buckets, but is not able to dielete files from S3 buckets.  Deleting data in S3 is done through the AWS cli.
+
+## Advanced Usage
+
+Documentation to aid in usingthe AWS cli is located here:
+[AWS CLI](https://sciwiki.fredhutch.org/compdemos/AWSCLI_WSL/).
+
+Note that the API endpoint to use with Fred Hutch's instance of Motuz is `https://motuz.fredhutch.org/api/` .
```
Comment (chrisequalsdev) [inline _pathways/path-migrating-data-from-fast-to-cloud.md:44]: Take this line out.

File: _pathways/path-migrating-data-from-fast-to-cloud.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Motuz is available on FHCC Campus or through the VPN at [https://motuz.fredhutch.org](https://motuz.fredhutch.org).
+Documentation on Motuz is located here:
+ - [Copying Files To Cloud Locations](https://sciwiki.fredhutch.org/compdemos/motuz/#copying-files-to-cloud-locations)
+
+Motuz can copy files into S3 buckets, but is not able to dielete files from S3 buckets.  Deleting data in S3 is done through the AWS cli.
+
+## Advanced Usage
+
+Documentation to aid in usingthe AWS cli is located here:
+[AWS CLI](https://sciwiki.fredhutch.org/compdemos/AWSCLI_WSL/).
```
Comment (chrisequalsdev) [inline _pathways/path-migrating-data-from-fast-to-cloud.md:42]: Would link to here instead: https://sciwiki.fredhutch.org/compdemos/AWSCLI_WSL/#working-with-s3-using-the-aws-cli

File: _pathways/path-migrating-data-from-fast-to-cloud.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+ - [AWS-Credentials](https://sciwiki.fredhutch.org/scicomputing/access_credentials/#amazon-web-services-aws)
+AWS credentials are assigned when each lab is given an AWS account.  If you need AWS Credentials, please send an email to helpdesk@fredhutch.org
+
+### Set Up Motuz to Move Data
+Motuz is the recommended tool to transfer data between Fred Hutch data storage locations.
+Motuz is available on FHCC Campus or through the VPN at [https://motuz.fredhutch.org](https://motuz.fredhutch.org).
+Documentation on Motuz is located here:
+ - [Copying Files To Cloud Locations](https://sciwiki.fredhutch.org/compdemos/motuz/#copying-files-to-cloud-locations)
+
+Motuz can copy files into S3 buckets, but is not able to dielete files from S3 buckets.  Deleting data in S3 is done through the AWS cli.
```
Comment (chrisequalsdev) [inline _pathways/path-migrating-data-from-fast-to-cloud.md:37]: Motuz won't delete from Fast, so if using Motuz to migrate data, a second step would be to manually delete data from Fast.

**Review by chrisequalsdev — COMMENTED** [review]
Added some comments.

File: _pathways/path-migrating-data-from-fast-to-cloud.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+If you store your data in `fh-PI-Name-eco` and decide to later share it with an external institute or lab, it must be moved from `fh-PI-Name-eco` to `fh-labname-eco-public`.
+
+If you need to access this data from the High Performance Compute Clusters, you will need to move it back to your `fh-PI-Lab` directory on Fast.
+
+## Steps
+
+### Acquire an AWS S3 account associated with your HutchNetID:
+Each employee will need AWS credentials to access S3 Economy Cloud.
+ - [AWS-Credentials](https://sciwiki.fredhutch.org/scicomputing/access_credentials/#amazon-web-services-aws)
+AWS credentials are assigned when each lab is given an AWS account.  If you need AWS Credentials, please send an email to helpdesk@fredhutch.org
```
Comment (vortexing) [inline _pathways/path-migrating-data-from-fast-to-cloud.md:29]: not really right? - AWS credentials are created user by user when the user requests them once a lab has been given an AWS account.  Right?

**Review by vortexing — COMMENTED** [review]
Do we have any documentation of the Motuz API?  I was unaware that that was even a thing, so maybe I just missed them.  

Also, do we still refer to AWS S3 storage as "Economy Cloud"?  Can we just write AWS S3?  I've heard from a LOAD of people that they get confused when people say "Economy" anything and wished IT would "just say AWS S3" b/c that's what is.  I'd vote for AWS S3 here too.

**Pull request merged**